OpenLiberty/guide-cloud-openshift-operator
# Merge qa to master: GH Actions - Daily build (#61)

* Prepare for daily builds

* Make files executable

* Add docker pull

* Test daily docker image

* clean package

* Get error stack traces

* Minor change to maven commands

* Revert change to mvn install command

* Clean install

* Remove offset commit in tests

* Test to see if issue is related to context or similar

* Add check for build level match

* Fix if-statement

* Update test.yml

* Update SystemServiceIT.java

* Update tests

* Add docker daily image testing

* Update SystemServiceIT.java

* Update SystemServiceIT.java

Co-authored-by: Austin Seto <austin.seto@ibm.com>
Co-authored-by: Gilbert Kwan <gkwan@ca.ibm.com>
